packages/catalog-model/src/kinds/ComponentV1beta1.ts

```diff
@@ -24,9 +24,6 @@ const KIND = 'Component';
 export interface ComponentV1beta1 extends Entity {
   apiVersion: typeof API_VERSION;
   kind: typeof KIND;
-  metadata: EntityMeta & {
-    name: string;
-  };
```

Comment:
this was unnecessary now that we ensure that `metadata` and `name` are present as part of all envelopes